src/khoj/interface/web/index.html

```diff
@@ -370,6 +375,17 @@
             max-width: 100;
         }
 
+        a {
+            color: #3b82f6;
+            text-decoration: none;
```

Comment:
Can we use links colors (like dark yellow) to stay aligned with the new lantern theme? The blue while nice, gives it a different feel

Reply:
Let me experiment with this. Our default yellow was resulting in an accessibility issue with contrast, but might work for a darker color (I think).

Reply:
I'm going to say let's leave this as-is for now, for two reasons.
1. The yellowish color from our branding poses certain accessibility constraints. We can think of what our "accent" color should be (or ask someone who can actually do UX 🤣).
2. Blue for links is an extremely standard UX pattern that everyone expects, even if they don't totally realize it. Hence, it's a good starting place.